py/desisim/scripts/quickquasars.py

```diff
@@ -462,8 +460,8 @@ def simulate_one_healpix(ifilename,args,model,obsconditions,decam_and_wise_filte
                     lyaforest=False, nocolorcuts=True,
                     noresample=True, seed=seed, south=issouth)
 
-        _meta['TARGETID'] = metadata['MOCKID']
-        _qsometa['TARGETID'] = metadata['MOCKID']
+        _meta['TARGETID'][:] = metadata['MOCKID']
+        _qsometa['TARGETID'][:] = metadata['MOCKID']
```

Comment:
Those two lines are what fixes the original bug?

Reply:
Yes, it is. Plus some other for DLA and BAL.